### PR TITLE
DZ60 keymap, ansi, split bs and rshift, 5 1u keys to right of space

### PR DESCRIPTION
**Pull request by orbitingorca**

### CODE REVIEW

File: keyboards/dz60/dz60.h [hunk 1]
```diff
@@ -162,4 +162,18 @@
 	{ K400,  K401, KC_NO,  K403, KC_NO, KC_NO,  K406, KC_NO, KC_NO, KC_NO,  K410,  K411,  K412,  K413,  K414 }  \
 }
 
+#define KEYMAP_60_ansi_split_bs_rshift_5x1u( \
```
Comment (mechmerlin) [inline keyboards/dz60/dz60.h]: Please use LAYOUT instead of KEYMAP

File: keyboards/dz60/info.json [hunk 2]
```diff
@@ -48,6 +48,10 @@
...
+    },
+    "KEYMAP_60_ansi_split_bs_rshift_5x1u": {
```
Comment (mechmerlin) [inline keyboards/dz60/info.json]: Please use LAYOUT instead of KEYMAP

**Comment by orbitingorca** [comment]
duh fixed.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**